app/src/main/java/com/babylon/wallet/android/presentation/account/history/HistoryScreen.kt

```diff
@@ -229,19 +231,25 @@ fun HistoryContent(
     }, onScrollEvent = {
         onScrollEvent(it)
     })
-    DefaultPullToRefreshContainer(
-        isRefreshing = state.isRefreshing,
+
+    val pullToRefreshState = rememberPullRefreshState(
+        refreshing = state.isRefreshing,
         onRefresh = onRefresh,
-        canRefresh = state.shouldEnableUserInteraction
+        refreshingOffset = 116.dp
```

Comment:
We could define this in RadixDimensions to reuse the value and easily change it if needed.

Reply:
I am not sure if `116.dp` is the correct offset. It made sense for the account screen but I am not sure it is right for the history screen. Seems to be copy pasted.